src/coupling_converters/coupling_converters_2d.jl

```diff
@@ -0,0 +1,35 @@
+# By default, Julia/LLVM does not use fused multiply-add operations (FMAs).
+# Since these FMAs can increase the performance of many numerical algorithms,
+# we need to opt-in explicitly.
+# See https://ranocha.de/blog/Optimizing_EC_Trixi for further details.
+@muladd begin
+#! format: noindent
+
+@doc raw"""
+    Coupling converter function for a system of two LinearScalarAdvectionEquation2D.
+
```

Comment:
```suggestion
@doc raw"""
    coupling_converter_heaviside_2d(x_0, c_0, s, equations)

Coupling converter function for a system of two LinearScalarAdvectionEquation2D.

```
Please add method signatures to dosctrings - also below

Does this really need to be specific to `LinearScalarAdvectionEquation2D`? To me, it feels like this is extremely specific to the example and may better live in the elixir file itself?

Reply:
Since there is an example in `examples/structured_2d_dgsem/elixir_advection_coupled_converter.jl` I removed it.